vnext/Desktop.DLL/react-native-win32.x64.def

```diff
@@ -46,6 +46,7 @@ EXPORTS
 ?typeName@dynamic@folly@@QEBAPEBDXZ
 ?updateProperties@ShadowNode@react@facebook@@UEAAX$$QEBUdynamic@folly@@@Z
 ?utf16ToUtf8@unicode@react@facebook@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@AEBV?$basic_string@_WU?$char_traits@_W@std@@V?$allocator@_W@2@@5@@Z
+?makeMemoryMappedBuffer@jsi@Microsoft@@YA?AV?$shared_ptr@VBuffer@jsi@facebook@@@std@@PEB_WI@Z
```

Comment:
is this export just for unit tests? do you want to add it for the uwp binary as well?

Reply:
That is my mistake. I'll add them to the uwp binary too.

---
In reply to: [320814285](https://github.com/microsoft/react-native-windows/pull/3059#discussion_r320814285) [](ancestors = 320814285)